frontend/src/__tests__/integration/pages/pipelines/PipelineDetails.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { test, expect } from '@playwright/test';
+import { navigateToStory } from '~/__tests__/integration/utils';
+
+test('Test topology renders', async ({ page }) => {
+  await page.goto(navigateToStory('pages-pipelines-pipelinedetails', 'default'));
+
+  // wait for page to load
+  await page.waitForSelector('text=test-pipeline');
+
+  // check the topology renders
+  expect(page.getByText('flip-coin')).toBeTruthy();
+  expect(page.getByText('random-num')).toBeTruthy();
+  expect(page.getByText('print-message')).toBeTruthy();
+
+  // check topology has tasks
+  await page.getByText('print-msg', { exact: true }).click();
+  await page.getByText('$(tasks.random-num.results.Output)').click();
```

Comment:
Why are we clicking on text that is not interactive?

Reply:
removed it